projects.yaml

```
@@ -1347,6 +1347,12 @@ projects:
   pypi_id: mkdocs-zip-folders
   labels: [plugin]
   category: site-conversion
+- name: mkdocs-exporter
+  mkdocs_plugin: mkdocs/exporter
```

Now we can change the name. And my other comment was going to be that we can actually register all the plugin entrypoints

```suggestion
  mkdocs_plugin: [exporter, exporter-pdf, exporter-extras]
```